src/ShowAll.js

```diff
@@ -1,41 +1,44 @@
-import React from "react";
-import FakeBookings from "./data/fakeBookings.json";
+import React, { useState } from "react";
 import Nights from "./Nights";
-const ShowAll = () => {
+const ShowAll = props => {
+  const [clicked, setClicked] = useState(false);
```

Comment:
This is a good attempt, but it looks like if you click on any row, then all of them become highlighted.

We'll look at a simple solution on Sunday, but the idea is to think carefully about how we could split up our components so that they can have their own state.

Reply:
Yes. I got this problem when I selected.